### ISSUE TITLE
Add "try it out" sections to GKE demos

### ISSUE THREAD

**Issue by dashpole** [GoogleCloudPlatform/opentelemetry-collector-builder-sample#20]
Here is an example for the redaction processor:

```json
{
  "resourceSpans": [
    {
      "resource": {
        "attributes": [
          {
            "key": "service.name",
            "value": {
              "stringValue": "redaction-demo"
            }
          }
        ]
      },
      "scopeSpans": [
        {
          "scope": {
            "name": "github.com/GoogleCloudPlatform/opentelemetry-collector-builder-samples/deploy/gke/redaction"
          },
          "spans": [
            {
              "traceId": "2499d13df7b35ae93dfa3d6ddc01da75",
              "spanId": "84f9e8929bb1cd5c",
              "parentSpanId": "",
              "name": "redacted span",
              "kind": "SPAN_KIND_INTERNAL",
              "startTimeUnixNano": "1658413463217165380",
              "endTimeUnixNano": "1658413463225568609",
              "attributes": [
                {
                  "key": "credit.card.number",
                  "value": {
                    "stringValue": "visa credit card number: 4111111111111111"
                  }
                }
              ],
              "status": {}
            }
          ]
        }
      ]
    }
  ]
}

```

**Comment by dashpole** [comment]
For these samples:

* https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample/tree/main/deploy/gke/redaction
* https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample/tree/main/deploy/gke/simple

Add a testdata.json file with some OTLP (like above) that demonstrates the collector's functionality (pii or a simple trace/metric/log)

Change the README.md to add instructions for creating a configmap from the testdata.json.

Change the manifest.yaml to add a volume + volumeMount for the configmap.

Change the otel-config.yaml to add an otlpjsonfile receiver that finds the test data.  Also, add the receiver to the appropriate pipelines at the end of otel-config.yaml.

Change the collector builder examples (both local and cloud build) to include the otlpjsonfile receiver.

**Comment by dashpole** [comment]
Customers can use the `kubectl logs` command to see the "logging" exporter output.

They can also navigate to https://console.cloud.google.com/traces/list to look for the trace ID from the input file

**Comment by psx95** [comment]
The sample json presented above does not seem to be a valid one for the `otlpjsonfile` receiver. The JSON is not supposed to be in a formatted way. 

I modified the tests in the receiver's package to output the file contents they were using and it looked something like - 
```
{"resourceSpans":[{"resource":{"attributes":[{"key":"resource-attr","value":{"stringValue":"resource-attr-val-1"}}]},"scopeSpans":[{"scope":{},"spans":[{"traceId":"","spanId":"","parentSpanId":"","name":"operationA","startTimeUnixNano":"1581452772000000321","endTimeUnixNano":"1581452773000000789","droppedAttributesCount":1,"events":[{"timeUnixNano":"1581452773000000123","name":"event-with-attr","attributes":[{"key":"span-event-attr","value":{"stringValue":"span-event-attr-val"}}],"droppedAttributesCount":2},{"timeUnixNano":"1581452773000000123","name":"event","droppedAttributesCount":2}],"droppedEventsCount":1,"status":{"message":"status-cancelled","code":"STATUS_CODE_ERROR"}},{"traceId":"","spanId":"","parentSpanId":"","name":"operationB","startTimeUnixNano":"1581452772000000321","endTimeUnixNano":"1581452773000000789","links":[{"traceId":"","spanId":"","attributes":[{"key":"span-link-attr","value":{"stringValue":"span-link-attr-val"}}],"droppedAttributesCount":4},{"traceId":"","spanId":"","droppedAttributesCount":4}],"droppedLinksCount":3,"status":{}}]}]}]}
```
Using json data in this format makes the receiver works as expected. 

The above json with removed whitespaces and formatting should work fine - 
```
{"resourceSpans":[{"resource":{"attributes":[{"key":"service.name","value":{"stringValue":"redaction-demo"}}]},"scopeSpans":[{"scope":{"name":"github.com/GoogleCloudPlatform/opentelemetry-collector-builder-samples/deploy/gke/redaction"},"spans":[{"traceId":"2499d13df7b35ae93dfa3d6ddc01da75","spanId":"84f9e8929bb1cd5c","parentSpanId":"","name":"redacted span","kind":"SPAN_KIND_INTERNAL","startTimeUnixNano":"1658413463217165380","endTimeUnixNano":"1658413463225568609","attributes":[{"key":"credit.card.number","value":{"stringValue":"visa credit card number: 4111111111111111"}}],"status":{}}]}]}]}
```

**Comment by dashpole** [comment]
Ah, that explains it. How id you find that out?  Maybe the otlpjson receiver needs better logging?

**Comment by psx95** [comment]
> Ah, that explains it. How id you find that out? Maybe the otlpjson receiver needs better logging?

Yes, it definitely needs proper logging, a simple log message stating "Improper formatting" could have pointed me in the right direction. 
I think I read somewhere in GitHub discussions that if parsing fails, it does not throw any errors. Not sure if its by design. 

I looked at their tests for the receiver. The tests were generating the content for the files so I outputted that content and used it as-is.
